upload/admin/language/cs-cz/extension/shipping/zasilkovna.php

```diff
@@ -126,7 +126,7 @@
 $_['error_invalid_weight']		= 'Váha musí být celé číslo.';
 $_['error_rules_overlapping']	= 'Pravidlo se překrývá s jiným pravidlem.';
 $_['error_duplicate_country_rule'] = 'Pravidlo pro tuto zemi je již definováno.';
-$_['error_guzzle_missing'] = 'Knihovna Guzzle není načtena, nebude možné aktualizovat dopravce pomocí cronu.';
+$_['error_disallowed_url_opening'] = 'Vaše PHP konfigurace neumožňuje otevírání externích URL. Nebude možné aktualizovat dopravce pomocí cronu.';
```

Comment:
Obvykle se uvádí, že jde o nastavení allow_url_fopen, aby to mohl eshopista rovnou někomu poslat. Raději bych to také uvedl.